TestAssets/TestPackages/dotnet-dependency-context-test/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-preview1-*",
+  "version": "1.0.0-preview2-*",
```

Comment:
Is the error because NuGet forces your `prerelease` version to start with a letter and not a number?  Maybe we could just stick a letter there, like `v`.  `1.0.0-v002700`.

This way we don't need to change these test assets every time we want to change the CLI's nuget prerelease.